client/src/main/java/io/pravega/client/stream/StreamConfiguration.java

```diff
@@ -83,12 +83,21 @@
     @EqualsAndHashCode.Exclude
     private final Set<String> tags;
 
+    /**
+     * API to return segment rollover size.
```

Comment:
What if the user is not interested in this? It will likely be defaulted to 0. In that case, I think it makes sense that `0` should be interpreted as "whatever is default on the server".

Let's update the javadoc to explain this. 

Please do this for KVT Config below too.